concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-1409	1409	"die Cardinalia von 1-10 mit ""Mal"""	siebenmal	seven times		
+Steinthal-1875-1565-1410	1410	"die Cardinalia von 1-10 mit ""Mal"""	achtmal	eight times		
+Steinthal-1875-1565-1411	1411	"die Cardinalia von 1-10 mit ""Mal"""	neunmal	nine times		
+Steinthal-1875-1565-1412	1412	"die Cardinalia von 1-10 mit ""Mal"""	zehnmal	ten times		
+Steinthal-1875-1565-1413	1413	noch einmal	noch einmal	once more		
+Steinthal-1875-1565-1414	1414	Wie oft?	Wie oft?	How often?		
+Steinthal-1875-1565-1415	1415	Wann?	Wann?	When?	1238	WHEN
+Steinthal-1875-1565-1416	1416	früher	früher	earlier		
+Steinthal-1875-1565-1417	1417	vor Alters	vor Alters	erstwhile		
+Steinthal-1875-1565-1418	1418	einst (eines Tages)	einst (eines Tages)	once	2914	ONCE (IN THE PAST)
+Steinthal-1875-1565-1419	1419	neulich	neulich	recently		
+Steinthal-1875-1565-1420	1420	zuerst	zuerst	at first	2915	AT FIRST
+Steinthal-1875-1565-1421	1421	noch nicht	noch nicht	not yet		
+Steinthal-1875-1565-1422	1422	schon	schon	already	2576	ALREADY
+Steinthal-1875-1565-1423	1423	immer	immer	always	1676	ALWAYS
+Steinthal-1875-1565-1424	1424	niemals	niemals	never	1656	NEVER
+Steinthal-1875-1565-1425	1425	täglich	täglich	daily		
+Steinthal-1875-1565-1426	1426	jetzt	jetzt	now	1376	NOW
+Steinthal-1875-1565-1427	1427	heute	heute	today	1283	TODAY
+Steinthal-1875-1565-1428	1428	morgen	morgen	tomorrow	1329	TOMORROW
+Steinthal-1875-1565-1429	1429	in diesem Jahre	in diesem Jahre	in this year	513	THIS YEAR
+Steinthal-1875-1565-1430	1430	im nächsten Jahre	im nächsten Jahre	in the next year	515	NEXT YEAR
+Steinthal-1875-1565-1431	1431	im vorigen Jahre	im vorigen Jahre	in the last year	514	LAST YEAR
+Steinthal-1875-1565-1432	1432	vor zwei, vor drei Jahren	vor zwei Jahren	two years ago		
+Steinthal-1875-1565-1433	1433	vor zwei, vor drei Jahren	vor drei Jahren	three years ago		
+Steinthal-1875-1565-1434	1434	in zwei Jahren	in zwei Jahren	in two years		
+Steinthal-1875-1565-1435	1435	in diesem Monat	in diesem Monat	in this month	2791	THIS MONTH
+Steinthal-1875-1565-1436	1436	im verflossenen nächsten Monat	im verflossenen Monat	in the last month	2792	LAST MONTH
+Steinthal-1875-1565-1437	1437	im verflossenen nächsten Monat	im nächsten Monat	in the next month	2793	NEXT MONTH
+Steinthal-1875-1565-1438	1438	soeben	soeben	just	84	JUST
+Steinthal-1875-1565-1439	1439	sogleich	sogleich	immediately	1963	IMMEDIATELY
+Steinthal-1875-1565-1440	1440	künftig	künftig	in the future		
+Steinthal-1875-1565-1441	1441	Welche Zeitrechnung hat ein Volk?	Welche Zeitrechnung hat ein Volk?	What calendar does a people use?		
+Steinthal-1875-1565-1442	1442	Wo?	Wo?	Where?	1237	WHERE
+Steinthal-1875-1565-1443	1443	Wohin?	Wohin?	Where to?	2928	WHITHER (WHERE TO)
+Steinthal-1875-1565-1444	1444	Woher?	Woher?	Where from?		
+Steinthal-1875-1565-1445	1445	hier	hier	here	136	HERE
+Steinthal-1875-1565-1446	1446	dort	dort	there	1937	THERE
+Steinthal-1875-1565-1447	1447	her	her	back	2910	BACK (DIRECTION)
```

Comment:
"her" rather means "towards the speaker".

Reply:
The prepositions without context, my favourite section of this list. I will unmap it. I just thought along the lines of "hin and her", i.e. "there and back".